app/models/orchid/facets.rb

```diff
@@ -34,4 +34,24 @@ def filter_facets
     end
   end
 
+  def section_facets(section)
+    if SECTIONS[section]["facets"].key?(I18n.locale.to_s)
+      facets = SECTIONS[section]["facets"][I18n.locale.to_s]
+    elsif SECTIONS[section]["facets"].key?("en")
```

Comment:
Question to ponder:  we default to english elsewhere if there is no language specified, but I wonder if we should check if there is a default language selected in the app_options, and if so, check for that before defaulting to english?

Reply:
Good idea! I like it

Reply:
Look good?
https://github.com/CDRH/orchid/pull/133/commits/024d56a96fc2d53eaa323d3cc3fc7e0a6971ca24